src/Set/DoctrineSetList.php

```diff
@@ -26,6 +26,8 @@ final class DoctrineSetList implements SetListInterface
      */
     public const DOCTRINE_COMMON_20 = __DIR__ . '/../../config/sets/doctrine-common-20.php';
 
+    public const DOCTRINE_COLLECTION_22 = __DIR__ . '/../../config/sets/doctrine-collection-22.php';
```

Comment:
add `@var string` like other

Reply:
also, if doctrine collection is part of doctrine/common, you can create a `DOCTRINE_COMMON_22` constant instead like above constant.

Reply:
ok, I force push the `@var string` declaration.

I am not sure that having that rule in DOCTRINE_COMMON_22 would not cause an issue: the `Order` class appears in the [release tagged 2.2.0](https://github.com/doctrine/collections/releases/tag/2.2.0). [Doctrine Common](https://packagist.org/packages/doctrine/common#2.13.3) require the version `^1.0`. So, as I understand, a project can have doctrine common >= 2.2 and doctrine collection < 2.2.0, and, in this case, replacing this would cause an error.

What do you think about that ?

Reply:
I see, that's confusing since it has `Doctrine\Common` subnamespace, but different package. I am ok with this, and let @TomasVotruba decide 👍